src/node/history.h

```diff
@@ -513,14 +513,18 @@ namespace ccf
       NodeId id_,
       tls::KeyPair& kp_,
       size_t sig_tx_interval_ = 0,
```

Comment:
Will we still use this interval to try to produce signatures on this `History`? I think if we can get away with it, we should avoid passing any of these timers to this store if it doesn't use them.

Reply:
No we won't, but I'm not sure what you have in mind?